tests/test_pipeline/test_pipeline.py

```diff
@@ -205,6 +206,26 @@ def test_forecast_with_intervals_other_model(base_forecast, model_class):
     )
 
 
+@pytest.mark.parametrize(
+    "model_class",
+    [
+        PredictionIntervalContextIgnorantAbstractModel,
+        PredictionIntervalContextRequiredAbstractModel,
+        NonPredictionIntervalContextIgnorantAbstractModel,
+    ],
```

Comment:
What about `NonPredictionIntervalContextRequiredAbstractModel`?

Reply:
I have updated the test. Basically it is just to test that `inverse_transform` calles in all 3 paths. 2 model specific cases and 1 for general one. We can add `NonPredictionIntervalContextRequiredAbstractModel`, but it will test the same as `NonPredictionIntervalContextIgnorantAbstractModel`.